pkg/scheduler/core/generic_scheduler.go

```diff
@@ -190,3 +191,24 @@ func (g *genericScheduler) chooseSpreadGroup(spreadGroup *util.SpreadGroup) []st
 	}
 	return feasibleClusters
 }
+
+func (g *genericScheduler) assignReplicas(clusters []*clusterv1alpha1.Cluster, replicaSchedulingStrategy *policyv1alpha1.ReplicaSchedulingStrategy, object *workv1alpha1.ObjectReference) ([]workv1alpha1.TargetCluster, error) {
+	if len(clusters) == 0 {
+		return nil, fmt.Errorf("no clusters available to schedule")
+	}
+	targetClusters := make([]workv1alpha1.TargetCluster, len(clusters))
+
+	if object.Replicas > 0 {
+		if replicaSchedulingStrategy.ReplicaSchedulingType == policyv1alpha1.ReplicaSchedulingTypeDuplicated {
```

Comment:
It seems like we need to check if replicaSchedulingStrategy == nil first?

Reply:
`if object.Replicas > 0 && replicaSchedulingStrategy != nil { ` 
like this ?

Reply:
I think it is ok.  I will run it locally.